config/settings.yml

```diff
@@ -353,11 +353,7 @@ claims_api:
   disability_claims_mock_override: false
   schema_dir: config/schemas
 
-# Settings for Redis
-# TODO(knkski): Move all redis settings here?
-redis:
-  host: localhost
-  port: 6379
+redis_url: 'redis://localhost:6379'
```

Comment:
This needs corresponding PRs in devops for the 4 environments as well, right @omgitsbillryan

Reply:
Saw it in the description -- my bad

Reply:
Yep. Probably even 2.

First, to add `redis_url`. Then once, dev, staging, & prod have all deployed with the `vets-api` code in this PR, a 2nd PR to devops that deletes the old setting.

Reply:
Oh! This will also affect anyone with a review instance who decides to merge the latest master of `vets-api` into their branch 🤔 ... I suppose it's probably ok to make an announcement in #vfs-platform-support